dev-python/bokeh/bokeh-2.0.2.ebuild

```diff
@@ -0,0 +1,108 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+# upstream hasn't tested python 3.8 fully
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Statistical and interactive HTML plots for Python"
+HOMEPAGE="https://bokeh.org/
+	https://github.com/bokeh/bokeh
+"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="examples"
+
+DISTUTILS_IN_SOURCE_BUILD=1
+
+# upstream authoritative dependencies
+# https://github.com/bokeh/bokeh/blob/master/conda.recipe/meta.yaml
+DEPEND="
+	dev-python/jinja[${PYTHON_USEDEP}]
+	dev-python/markupsafe[${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	dev-python/packaging[${PYTHON_USEDEP}]
+	dev-python/pillow[${PYTHON_USEDEP}]
+	dev-python/pyparsing[${PYTHON_USEDEP}]
+	dev-python/python-dateutil[${PYTHON_USEDEP}]
+	dev-python/pyyaml[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+	dev-python/typing-extensions[${PYTHON_USEDEP}]
+	>=www-servers/tornado-5[${PYTHON_USEDEP}]
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	${DEPEND}
+	sys-apps/ripgrep[pcre]
+	test? (
+		dev-python/beautifulsoup:4[${PYTHON_USEDEP}]
+		dev-python/boto[${PYTHON_USEDEP}]
+		dev-python/colorama[${PYTHON_USEDEP}]
+		dev-python/flaky[${PYTHON_USEDEP}]
+		dev-python/ipython[notebook,${PYTHON_USEDEP}]
+		dev-python/pandas[full-support,${PYTHON_USEDEP}]
```

Comment:
This causes dependencies for this package to be nonmergeable without some great amounts of handwork. Portage cannot handle it no matter what I do. Is this dependency **really** required? The `full-support` I mean. 

If yes, I'm starting to incline towards not having *bokeh* in ::gentoo at all. It seems broken upstream and proves to be a lot of work to get manageable for us. And I don't believe in the long run things will get easier. Sorry for all the work you've put to this so far, but this is how I feel.

Waiting for your input.

Reply:
Ah. No I don't think it is.  
It might have been added because I was frustrated and adding all the possible options.   

I think ripgrep can also be removed.  
I'm going to check out in a new clean environment.

Reply:
ok, i've removed ripgrep (was for integration tests, we are not doing those) and also remove full-support form pandas (we only need pandas dataframe for fixtures in tests)

everything should be working now :smiley_cat: